packages/cli/src/UserManagement/email/NodeMailer.ts

```diff
@@ -16,6 +16,11 @@ export class NodeMailer {
 			secure: config.getEnv('userManagement.emails.smtp.secure'),
 		};
 
+		if (config.getEnv('userManagement.emails.smtp.debug')) {
```

Comment:
Passing in our logger seems like a good idea, I wanted to keep the SMTP debug split as it is very chatty with the entire handshake being outputted so you may not want to have all of that clogging up your debug logs and you may not want the other debug logs getting in the way of trying to debug an SMTP issue.

Will give it a go and see how it looks.

Reply:
another option could be to use the [`debug` npm package](https://www.npmjs.com/package/debug) with the scope `n8n:smtp`. 
I plan to move all `verbose` and `debug` logging to that package in the near future, so this could be a good place to start with that.